src/dsp_tools/commands/xml_validate/xml_validate.py

```diff
@@ -13,25 +15,38 @@
 from dsp_tools.utils.xml_validation import validate_xml
 
 
-def _get_data_info_from_file(file: Path, namespace: str) -> tuple[ProjectInformation, DataRDF]:
-    cleaned_root = _parse_and_clean_file(file, namespace)
+def _get_project_ontos(onto_con: OntologyConnection) -> Graph:
+    all_ontos = onto_con.get_ontologies()
+    g = Graph()
+    for onto in all_ontos:
+        og = Graph()
+        og.parse(data=onto, format="ttl")
+        g += og
+    return g
+
+
+def _get_data_info_from_file(file: Path, api_url: str) -> tuple[ProjectInformation, DataRDF]:
+    cleaned_root = _parse_and_clean_file(file, api_url)
     deserialised: ProjectDeserialised = deserialise_xml(cleaned_root)
     rdf_data: DataRDF = make_data_rdf(deserialised.data)
     return deserialised.info, rdf_data
 
 
-def _parse_and_clean_file(file: Path, namespace: str) -> etree._Element:
+def _parse_and_clean_file(file: Path, api_url: str) -> etree._Element:
     root = parse_xml_file(file)
     root = remove_comments_from_element_tree(root)
     validate_xml(root)
     root = transform_into_localnames(root)
-    return _replace_namespaces(root, namespace)
+    return _replace_namespaces(root, api_url)
 
 
-def _replace_namespaces(root: etree._Element, ontology_namespace: str) -> etree._Element:
+def _replace_namespaces(root: etree._Element, api_url: str) -> etree._Element:
     with open("src/dsp_tools/resources/xml_validate/replace_namespace.xslt", "rb") as xslt_file:
         xslt_data = xslt_file.read()
+    shortcode = root.attrib["shortcode"]
```

Comment:
This is not ideal, but I was unable to change the xslt file to retrieve the information and use it. In my opinion we can leave it as is for now and try and clean it later.